src/main/java/de/tum/in/www1/artemis/domain/Rating.java

```diff
@@ -0,0 +1,70 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+/**
+ * A Rating.
+ */
+@Entity
+@Table(name = "result_rating")
+@JsonInclude(Include.NON_EMPTY)
+public class Rating implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "rating")
+    private Integer rating;
+
+    @OneToOne()
```

Comment:
Can you remove the unnecessary parentheses here?

Reply:
Done